hostcheck@schorschii/files/hostcheck@schorschii/desklet.js

```diff
@@ -160,7 +161,7 @@ MyDesklet.prototype = {
 		let label_size_x = this.defaultWidth * this.scale_size * global.ui_scale;
 		let label_size_y = this.defaultHeight * this.scale_size * global.ui_scale
 		// modify label
-		let statusString = this.statusTagString + "\n" + this.host;
+		let statusString = this.statusTagString + " / " + this.interval + "s\n" + this.host;
```

Comment:
@Gujenczek Thanks for these improvements. I only have one thing to say, can you please make the "s" translatable and put the interval in brackets (no slash)?